conditionals.md

```diff
@@ -0,0 +1,227 @@
+## Conditionals Statements
```

Comment:
Let's use a first-level heading here, that is, `#` instead of `##` since it's the topic. 
I would also recommend you call it `Conditional Statement in Javascript`, some programming language conditional statement is slightly different, it's better to make it known that we taking about javascript for this article.

Reply:
Thanks, I guess JavaScript Conditional Statement is also fine.